pkg/tso/allocator_manager.go

```diff
@@ -134,17 +132,20 @@ type AllocatorManager struct {
 func NewAllocatorManager(
 	m *member.Member,
 	rootPath string,
-	cfg config,
+	enableLocalTSO bool,
```

Comment:
passing all tso configuration fields here seems to be a litter weird, but we can refine later.

Reply:
Yes, we can define an interface later to avoid it.